api/src/main/java/org/pmiops/workbench/db/model/DbDemographicSurvey.java

```diff
@@ -174,6 +176,34 @@ public void setRaceEnum(List<Race> raceList) {
         raceList.stream().map(DemographicSurveyEnum::raceToStorage).collect(Collectors.toList());
   }
 
+  @ElementCollection(fetch = FetchType.LAZY)
+  @CollectionTable(
+      name = "demographic_survey_gender",
+      joinColumns = @JoinColumn(name = "demographic_survey_id"))
+  @Column(name = "gender")
+  public List<Short> getGender() {
+    return gender;
+  }
+
+  public void setGender(List<Short> gender) {
+    this.gender = gender;
+  }
+
+  @Transient
+  public List<Gender> getGenderEnum() {
+    if (gender == null) return null;
+    return this.gender.stream()
+        .map(DemographicSurveyEnum::genderFromStorage)
+        .collect(Collectors.toList());
+  }
+
+  public void setGenderEnum(List<Gender> genderlist) {
```

Comment:
Should we be using List, or would Set be better? I.e. does order matter? I don't think we want duplicate entries.